src/commands/logic/run/test.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { CancellationTokenSource } from '@salesforce/apex-node';
+import {
+  arrayWithDeprecation,
+  Flags,
+  loglevel,
+  orgApiVersionFlagWithDeprecations,
+  requiredOrgFlagWithDeprecations,
+  SfCommand,
+} from '@salesforce/sf-plugins-core';
+import { Messages } from '@salesforce/core';
+import { codeCoverageFlag, resultFormatFlag } from '../../../flags.js';
+import { TestRunService, TestLevelValues, RunCommandResult, TestRunConfig } from '../../../shared/TestRunService.js';
+
+Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
+const messages = Messages.loadMessages('@salesforce/plugin-apex', 'runlogictest');
+const commonFlags = Messages.loadMessages('@salesforce/plugin-apex', 'runtest');
+
+const exclusiveTestSpecifiers = ['class-names', 'suite-names', 'tests', 'test-category'];
+export default class Test extends SfCommand<RunCommandResult> {
+  public static readonly summary = messages.getMessage('summary');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessages('examples');
+  public static readonly deprecateAliases = true;
+  public static readonly aliases = ['force:logic:test:run'];
```

Comment:
Same comment as in your other PR: I don't think we want this alias, because "force logic test run" doesn't exist.  but check with the CLI team.